### PR TITLE
tilde character wasn't playing nice when assigning the APP_BUNDLE_PATH.

### PR DESCRIPTION
**Pull request by sebglazebrook**
Updated documentation to use the full path.

### CODE REVIEW

**Comment by jmoody** [comment]
@sebglazebrook 

Have you seen this problem in the wild?  I don't think that I have.

**Comment by sebglazebrook** [comment]
@jmoody 
Hey,

Here's an example:

If in file `01_launch.rb` I have this set:
`APP_BUNDLE_PATH = "#{ENV['HOME']}/Library/Developer/Xcode/DerivedData/??/Build/Products/Debug-iphonesimulator/??.app"`

Then everything works fine.

If in file `01_launch.rb` I have this set:
`APP_BUNDLE_PATH="~/Library/Developer/Xcode/DerivedData/??/Build/Products/Debug-iphonesimulator/??.app"`

I get the following error:
```
no implicit conversion of nil into String (TypeError)
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop/lipo.rb:118:in `join'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop/lipo.rb:118:in `binary_path'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop/lipo.rb:84:in `info'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop/lipo.rb:65:in `expect_compatible_arch'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop/core.rb:116:in `expect_compatible_simulator_architecture'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop/core.rb:202:in `run_with_options'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/run_loop-1.2.9/lib/run_loop.rb:77:in `run'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.13.0/lib/calabash-cucumber/launcher.rb:761:in `block in new_run_loop'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.13.0/lib/calabash-cucumber/launcher.rb:759:in `times'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.13.0/lib/calabash-cucumber/launcher.rb:759:in `new_run_loop'
  /opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.13.0/lib/calabash-cucumber/launcher.rb:630:in `relaunch'
  /Users/sebastianglazebrook/code/app-name/features/support/01_launch.rb:29:in `Before'
```

**Comment by sebglazebrook** [comment]
Also I'm guess people using `/bin/sh` will have issues.

That's not the case in this case thought.

File: calabash-cucumber/ENVIRONMENT_VARIABLES.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ Paths or values with spaces need double or single quotes.
 #### Example: Quoting values with spaces.
 
 ```
-APP_BUNDLE_PATH="~/path with/a spaces/in it"   # Correct!
-APP_BUNDLE_PATH=~/path with/a spaces/in it     # Incorrect.
+APP_BUNDLE_PATH="Users/adamant/path with/a spaces/in it"   # Correct!
+APP_BUNDLE_PATH=Users/adam ant/path with/a spaces/in it     # Incorrect.
```
Comment (jmoody) [inline calabash-cucumber/ENVIRONMENT_VARIABLES.md]: +1 for the Adam Ant reference.

File: calabash-cucumber/scripts/launch.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@
 require 'calabash-cucumber/launcher'
 
 
-# APP_BUNDLE_PATH = "~/Library/Developer/Xcode/DerivedData/??/Build/Products/Calabash-iphonesimulator/??.app"
+# APP_BUNDLE_PATH = "#{ENV['HOME']}/Library/Developer/Xcode/DerivedData/??/Build/Products/Calabash-iphonesimulator/??.app"
```
Comment (jmoody) [inline calabash-cucumber/scripts/launch.rb]: Wow! This file has got to go.  Ancient!

**Comment by jmoody** [comment]
@sebglazebrook Thanks!

**Pull request merged**